core/src/main/java/org/geysermc/geyser/translator/inventory/item/nbt/EnchantmentTranslator.java

```diff
@@ -127,6 +137,14 @@ private CompoundTag remapEnchantment(CompoundTag tag) {
 
         Enchantment enchantment = Enchantment.getByJavaIdentifier(((StringTag) javaEnchId).getValue());
         if (enchantment == null) {
+            if (javaEnchId.getValue().equals("minecraft:sweeping")){
```

Comment:
Check formatting with brackets here.

Reply:
oh god i always miss that
can is somehow make intelliJ suggest a code style or so